Java/crypto/src/main/java/com/breadwallet/crypto/blockchaindb/apis/bdb/TransactionApi.java

```diff
@@ -37,13 +35,19 @@
     private final BdbApiClient jsonClient;
     private final ExecutorService executorService;
 
-    public TransactionApi(BdbApiClient jsonClient, ExecutorService executorService) {
+    public TransactionApi(BdbApiClient jsonClient,
+                          ExecutorService executorService) {
         this.jsonClient = jsonClient;
         this.executorService = executorService;
     }
 
-    public void getTransactions(String id, List<String> addresses, UnsignedLong beginBlockNumber, UnsignedLong endBlockNumber,
-                                boolean includeRaw, boolean includeProof, @Nullable Integer maxPageSize,
+    public void getTransactions(String id,
+                                List<String> addresses,
+                                UnsignedLong beginBlockNumber,
+                                UnsignedLong endBlockNumber,
+                                boolean includeRaw,
+                                boolean includeProof,
+                                @Nullable Integer maxPageSize,
```

Comment:
This will be in conflict with CORE-525 where begin/endBlockNumber are optional.  Should we resolve it now, here?